lunatrace/cli/cmd/lunatrace/main.go

```diff
@@ -50,88 +45,65 @@ func main() {
 		util.RemoveCleanupDirs()
 	})
 
-	setGlobalBoolFlags := func(c *cli.Context) error {
-		for flag := range globalBoolFlags {
-			if c.IsSet(flag) {
-				globalBoolFlags[flag] = true
-			}
-		}
-		return nil
+	enabledLunaTraceAsset := []types.CliAssetCmdConfig{
+		{
+			Usage:        "Collect a snapshot for a repository.",
```

Comment:
Remote repository* If I saw this I 2ould assume it's designed to be used locally